package-system/vulkan-validationlayers/build_vulkan_validation_windows.cmd

```diff
@@ -0,0 +1,33 @@
+@rem #
+@rem # Copyright (c) Contributors to the Open 3D Engine Project.
+@rem # For complete copyright and license terms please see the LICENSE at the root of this distribution.
+@rem # 
+@rem # SPDX-License-Identifier: Apache-2.0 OR MIT
+@rem #
+
+@rem # note that we explicitly turn off the compilation of all features that rely on 3rd Party Libraries
+@rem # except the ones we want.  This prevents the cmake build system from automatically finding things
+@rem # if they happen to be installed locally, which we don't want.
+
+@rem # cmake expects fowardslashes:
+set "DOWNLOADED_PACKAGE_FOLDERS=%DOWNLOADED_PACKAGE_FOLDERS:\=/%"
+
+@rem cd temp/build
+
+@rem release build
+python.exe temp/src/scripts/update_deps.py --dir temp/src/external --arch x64 --config release
```

Comment:
If you want to use the same python that pull_and_build_from_git was launched you can use the env var %PYTHON_BINARY%
pull_and_build_from_git.py sets this env var to the python binary that was used to invoke it in all child processes.  This ensures that even if python.exe is not on the path, or its the wrong version, you will be invoked with the approrpiate python3 executable.  It does need python3, right?  Its not a python2 script?